tests/utils/parseropts/parseropts.py

```diff
@@ -127,6 +127,10 @@ def pytest_configure(config):
         "markers",
         "conversion: mark test to run only when --test-conversion cli parameter is provided",
     )
+    config.addinivalue_line(
+        "markers",
+        "not_for_machine: Fixture that enables use of `not_for_machine(machine-name)` marker.",
```

Comment:
By looking at the way we describe the other markers, this one is estrange. I would have expected a direct description
```suggestion
        "not_for_machine: exclude only for the given machine.",
```